lib/models/Veeam.ts

```diff
@@ -72,9 +73,9 @@ export class VeeamCapacityInfo {
 
     static toBigInt(capacity: VeeamSOSApiSchema['CapacityInfo']): VeeamSOSApiSchema['CapacityInfo'] {
```

Comment:
the return type is not correct : it now has only `bigint` only

Reply:
As we also have the lastModified which is not a big int => keeping the VeeamSOSApiSchema['CapacityInfo'] seems fair...

Reply:
the point of typing is to allow the compiler to do the work : if we type too broadly, then typing becomes useless, we have no benefit anymore... The whole point of this function is to convert to bigint, so best to specify the actual type...
```suggestion
    static toBigInt(capacity: VeeamSOSApiSchema['CapacityInfo']): { Capacity: BigInt; Available: BigInt; Used: BigInt; LastModified?: string } {
```